tests/datastore-tests.ts

```diff
@@ -1,19 +1,22 @@
 import { MemoryDataStore } from '../src/datastore/memory-store';
-import { DbBlock } from '../src/datastore/common';
-import { PgDataStore, cycleMigrations } from '../src/datastore/postgres-store';
+import { DbBlock, DbTx, DbTxTypeId } from '../src/datastore/common';
+import { PgDataStore, cycleMigrations, runMigrations } from '../src/datastore/postgres-store';
 
 describe('in-memory datastore', () => {
   let db: MemoryDataStore;
+
   beforeAll(() => {
     db = new MemoryDataStore();
   });
+
```

Comment:
Don't we want to initialise on every test? `beforeEach()`

Reply:
Good call, fixed